### ISSUE TITLE
Firewall Drop

### ISSUE THREAD

**Issue by bsharin** [wazuh/wazuh#1877]
Hi,

The **firewall-drop.sh** still is not working in the latest release.

- **ossec.conf** defines firewall-drop as **firewall-drop.sh** as it has existed in previous releases, but **firewall-drop** was removed from the active-response directory and moved the the firewalls subdirectory as a new filename, **default-firewall-drop.sh**

```
<command>
    <name>firewall-drop</name>
    <executable>firewall-drop.sh</executable>
    <expect>srcip</expect>
    <timeout_allowed>yes</timeout_allowed>
</command>
```

Please move **firewall-drop.sh** back to the active-response directory to maintain consistency with ossec.conf and to allow for backwards compatibility.

References:
https://github.com/wazuh/wazuh/issues/1103
https://github.com/wazuh/wazuh/issues/1356
https://github.com/wazuh/wazuh/issues/1282

Thanks,

Blake

**Comment by Skeptor** [comment]
Hi @bsharin 
I have tested the active-response with the default command configuration and the active-response configuration I will post you below and it works fine when a sshd alert is triggered:
```
<active-response>
    <disabled>no</disabled>
    <command>firewall-drop</command>
    <location>all</location>
    <level>3</level>
    <rules_group>sshd</rules_group>
    <timeout>10</timeout>
  </active-response>
```

I did the test in Ubuntu 18, with a Wazuh 3.7 manager.

This is my active-response alert:
```
** Alert 1542211747.1259096: - ossec,active_response,pci_dss_11.4,gpg13_4.13,gdpr_IV_35.7.d,
2018 Nov 14 17:09:07 (c7-agent-12) 192.168.1.56->/var/ossec/logs/active-responses.log
Rule: 601 (level 3) -> 'Host Blocked by firewall-drop.sh Active Response'
Src IP: 10.0.2.2
Wed Nov 14 16:09:06 UTC 2018 /var/ossec/active-response/bin/firewall-drop.sh add - 10.0.2.2 1542211746.1258168 5715
script: firewall-drop.sh
type: add

** Alert 1542211758.1259590: - ossec,active_response,pci_dss_11.4,gpg13_4.13,gdpr_IV_35.7.d,
2018 Nov 14 17:09:18 (c7-agent-12) 192.168.1.56->/var/ossec/logs/active-responses.log
Rule: 602 (level 3) -> 'Host Unblocked by firewall-drop.sh Active Response'
Src IP: 10.0.2.2
Wed Nov 14 16:09:17 UTC 2018 /var/ossec/active-response/bin/firewall-drop.sh delete - 10.0.2.2 1542211746.1258168 5715
script: firewall-drop.sh
type: delete
```

Could you please tell me what environment do you have? There we have any difference between our ossec.conf?

Regards,
Pablo.

**Comment by bsharin** [comment]
Hi @Skeptor,

Thanks for looking into this.  I am running Ubuntu 16.04.  My mistake for not testing myself first.  I saw the repo 
![image](https://user-images.githubusercontent.com/24966547/48502548-c9a43180-e80d-11e8-928c-464d71c8d86a.png)
and assumed that is was left out.  I just upgraded my development environment to 3.7 and **firewall-drop.sh** is indeed included in at the proper path **/var/ossec/active-response/bin/** and I tested to make sure active-repsonse gets properly triggered and it does.  I will close this issue.  Thanks again.

Blake
